drivers/include/sx127x.h

```diff
@@ -92,8 +92,22 @@ extern "C" {
 #define SX127X_IRQ_DIO3                  (1<<3)  /**< DIO3 IRQ */
 #define SX127X_IRQ_DIO4                  (1<<4)  /**< DIO4 IRQ */
 #define SX127X_IRQ_DIO5                  (1<<5)  /**< DIO5 IRQ */
+/** @} */
+
+/**
+ * @defgroup drivers_sx127x_config     Semtech SX1272 and SX1276 driver compile configuration
+ * @ingroup config
+ * @{
+ */
+
+/**
+ * @brief Enable pull down mode on DIOx Pins.
+ *
+ * Pull Down the GPIO PINS .
+ * By default the pins are in Push-Pull configuration.
```

Comment:
This parameter does not enable pull-down mode, it is the mode it will be applied to the GPIOs. I think it should be something like:
```suggestion
 * @brief GPIO mode of DIOx Pins.
 *
 * By default the pins are configured as inputs with pull-down resistor.
```